nominees/awareness-sms.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Awareness SMS",
+  "description": "Node.js wrapper for Africa’s Talking SMS API. Allows government entities to send mass SMS to text the public critical COVID-19 related information.",
+  "website": "https://github.com/africa-covid-19-response-toolkit/awareness-sms",
```

Comment:
@318anushka kindly update the website field with the umbrella `[websiteURL](https://ethiocovid19rt.com/)` rather than the Github repo.